<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dalee2</class>
 <widget class="QWidget" name="Dalee2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Вперед</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>350</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Налево</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>631</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;style&gt; p {text-indent: 50px} &lt;/style&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Вы выбрали правильный ответ. Перед вами&lt;br/&gt;открывается дверь. Пройдя дальше вы вздыхаете с&lt;br/&gt;облечением, так как в одно мгновенье могли лишиться&lt;br/&gt;собственной жизни.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Пройдя дальше вы натыкаетесь на еще одну&lt;br/&gt;развилку. В данной ситуации у вас два пути. Повернуть&lt;br/&gt;налево и пройти вперед. Не желая тратить много времени&lt;br/&gt;на обдумывания вы решаете пойти дальше.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pict">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../PICT/pict2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_pict</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
